agents/base.py

```
@@ -96,6 +96,90 @@ def truncate_for_reply(
+        "rows": [[str(cell) for cell in row] for row in rows],
+    }
+
+
+def links_block(
+    links: List[Dict[str, str]], label: Optional[str] = None
+) -> Dict[str, Any]:
+    """A list of sources or destinations."""
+    return {
+        "type": "links",
+        "label": label,
+        "links": [{"label": str(l["label"]), "url": str(l["url"])} for l in links],
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)